.gitlab/scripts.yml

```diff
@@ -45,6 +45,7 @@
         -DGINKGO_BUILD_HWLOC=${BUILD_HWLOC}
         -DGINKGO_BUILD_PAPI_SDE=${BUILD_PAPI_SDE}
         -DGINKGO_BUILD_TESTS=ON -DGINKGO_BUILD_EXAMPLES=ON
+        -DGINKGO_BUILD_EXTENSION=${BUILD_EXTENSION}
```

Comment:
Does it need to be optional? If so, `EXTENSION` is not a very descriptive name, maybe something like `CONFIG`